dask/dataframe/tests/test_arithmetics_reduction.py

```diff
@@ -1346,3 +1346,10 @@ def test_moment():
     ddf = dd.from_pandas(df, npartitions=2)
 
     assert_eq(stats.moment(ddf, 2, 0), scipy.stats.moment(df, 2, 0))
+
+
+def test_datetime_series_mean():
```

Comment:
Can you add a DataFrame test too, both all-datetime columns and a mix of datetime and non datetime?

Reply:
Have you had any luck with a mixed-dtype DataFrame (some datetimes, some integer, etc.)? I'm concerned about the implementation in this case.

Reply:
added these additional lines to the test and it is passing
```
pdf[1] = 4
ddf = dd.from_pandas(pdf, npartitions=1)
assert_eq(ddf.mean(), pdf.mean())
assert_eq(ddf.mean(axis=1), pdf.mean(axis=1))
```

Reply:
Ahh, pandas drops datetimes from DataFrame.mean, see https://github.com/pandas-dev/pandas/issues/28108.

I don't think the implementation hear will gracefully handle things if / when that changes in pandas, but it's probably fine to just match pandas for now.